httpclient/src/test/java/feign/httpclient/ApacheHttpClientTest.java

```diff
@@ -33,11 +33,13 @@
 import feign.RequestLine;
 import feign.Response;
 
-import static feign.Util.UTF_8;
-import static feign.assertj.MockWebServerAssertions.assertThat;
 import static java.util.Arrays.asList;
+
 import static org.junit.Assert.assertEquals;
 
+import static feign.Util.UTF_8;
```

Comment:
similarly, please revert the import change so we know what's different (if anything)